### PR TITLE
Topic page for Equity

### PR DESCRIPTION
**Pull request by beets**
- delete climate topic page
- a few small updates to poverty page
- add a new tile type: description

### CODE REVIEW

File: server/config/topic_page/equity/USA.textproto [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+    stat_var_override {
+      stat_var: "UnemploymentRate_Person_Female"
+      unit: "%"
+    }
+    stat_var_override {
+      stat_var: "UnemploymentRate_Person_Male"
+      unit: "%"
+    }
+  }
+  # right_tiles {
```
Comment (chejennifer) [inline server/config/topic_page/equity/USA.textproto:71]: did you mean to keep this tile commented out?

File: server/config/topic_page/poverty/USA.textproto [hunk 2]
```diff
@@ -310,4 +332,43 @@ blocks {
...
+    type: LINE
+    title: "Education attainment for households in poverty"
+  }
+  right_tiles {
+    type: BAR
+    title: ""
+  }
+}
+
+# blocks {
```
Comment (chejennifer) [inline server/config/topic_page/poverty/USA.textproto:365]: comment what this commented out section is for?

**Review by chejennifer — APPROVED** [review]
nice!

File: server/config/topic_page/equity/USA.textproto (hunk 1 above)
Comment (beets) [reply server/config/topic_page/equity/USA.textproto]: deleted, and added a comment instead.

File: server/config/topic_page/poverty/USA.textproto (hunk 2 above)
Comment (beets) [reply server/config/topic_page/poverty/USA.textproto:366]: done

**Comment by beets** [comment]
thanks for the review!

**Pull request merged**